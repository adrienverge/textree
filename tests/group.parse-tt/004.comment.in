// comment 1
element1
// comment 2 must not break element1 because of indent
  child
  //last comment no leading space
