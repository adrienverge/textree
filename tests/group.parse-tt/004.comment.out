{"type":"comment","value":"// comment 1"}
{"type":"start","name":"element1"}
{"type":"comment","value":"// comment 2 must not break element1 because of indent"}
{"type":"start","name":"child"}
{"type":"comment","value":"//last comment no leading space"}
{"type":"end"}
{"type":"end"}
